depobs/scanner/clients/npm_registry.py

```diff
@@ -7,7 +7,7 @@
 import logging
 
 from depobs.scanner.pipelines.util import exc_to_str
-from depobs.scanner.serialize_util import grouper
+from depobs.util.serialize_util import grouper
```

Comment:
do we need to move `serialize_util` to `util` to combine the db schemas?

I think `util` and `misc` namespaces are kind of an anti pattern (I realize that I added them). We have a graph_util package too.